packages/experiments/src/components/signals/Signal.scss

```diff
@@ -9,7 +9,7 @@
 
   .dark &,
   .dark.selected {
-    color: $ms-color-white;
+    color: $ms-color-yellowDark;
```

Comment:
Look at the `SignalsPage` and switch to 'Is Dark'. Not sure we want them all to be yellow!
"Dark mode" for Signals makes them render white against a dark background for a media tile.